src/v3/test/integration/identify-with-password.test.tsx

```diff
@@ -44,15 +51,9 @@ describe('identify-with-password', () => {
     expect(passwordEl).toHaveAccessibleDescription(passwordHint);
   });
 
-  it('renders the loading state first', async () => {
-    const { container, findByLabelText } = await setup({ mockResponse });
-    await findByLabelText('Processing...');
-    expect(container).toMatchSnapshot();
-  });
-
   it('renders form with focus', async () => {
-    const { container, findByLabelText } = await setup({ mockResponse });
-    const inputEle = await findByLabelText(/Username/) as HTMLInputElement;
```

Comment:
These tests needed to be reordered due to side-effects caused by other tests running earlier. I will followup with a separate fix since the proper resolution is out of scope